src/test/java/org/apache/sysml/test/integration/functions/paramserv/ParamservFuncTest.java

```diff
@@ -32,7 +32,9 @@
 	private static final String TEST_NAME4 = "paramserv-wrong-type-args";
 	private static final String TEST_NAME5 = "paramserv-wrong-args";
 	private static final String TEST_NAME6 = "paramserv-wrong-args2";
-	private static final String TEST_NAME7 = "paramserv-ipa-test";
+	private static final String TEST_NAME7 = "paramserv-wrong-list-test";
+	private static final String TEST_NAME8 = "paramserv-nn-test";
+	private static final String TEST_NAME9 = "paramserv-minimum-version";
```

Comment:
When I executed these tests, test 8 and 9 were still failing - could you please have a look?